src/management/OrganizationsManager.js

```diff
@@ -314,6 +317,7 @@ OrganizationsManager.prototype.getEnabledConnection = function(params, callback)
  *   }
  * });
  *
+ * @param   {Object}    params                Organization parameters
```

Comment:
The method signature doesnt include the first parameter:

![image](https://user-images.githubusercontent.com/2146903/135080309-b46280a1-46cb-4439-8c72-782685276366.png)

It should be `addEnabledConnection(params, data, cb)` and not `addEnabledConnection(data, cb)`